scalardb-test/src/main/java/kelpie/scalardb/transfer/graphql/ThreadLocalCookieJar.java

```diff
@@ -0,0 +1,43 @@
+package kelpie.scalardb.transfer.graphql;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import okhttp3.Cookie;
+import okhttp3.CookieJar;
+import okhttp3.HttpUrl;
+import org.jetbrains.annotations.NotNull;
+
+/** A {@code CookieJar} implementation that manages cookies for each thread. */
+public class ThreadLocalCookieJar implements CookieJar {
```

Comment:
Added `ThreadLocalCookieJar` because we need to mange cookie for the session affinity case.